Makefile

```diff
@@ -132,7 +132,8 @@ BASE_PACKAGE_NAME := github.com/dapr/dapr
 LOGGER_PACKAGE_NAME := github.com/dapr/kit/logger
 
 # Comma-separated list of features to enable
-ENABLED_FEATURES ?=
+# TODO: @ItalyPaleAle remove when ServiceInvocationStreaming is finalized
+ENABLED_FEATURES ?= ServiceInvocationStreaming
```

Comment:
I need to remove this before the PR is merged - for now it's here so E2E tests run with this to validate the PR